deployment/v3/external/iam/keycloak_init.sh

```diff
@@ -9,7 +9,7 @@ fi
 
 function initialize_keycloak() {
   NS=keycloak
-  CHART_VERSION=12.0.1-B3
+  CHART_VERSION=12.0.2
```

Comment:
we can point to the 12.0.1  helm chart


as the same code is present on both develop & 1.2.0.1 branches of keycloak-init